<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="mainStack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>800</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="consentForm">
    <widget class="QLabel" name="headerLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>1600</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>The following experiment was granted ethical approval by the UCL Psychology and Langage Sciences ethics chair.</string>
     </property>
    </widget>
    <widget class="QLabel" name="informationLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>1400</width>
       <height>120</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This experiment will involve making decisions in response to presented stimuli. You will be given detailed information before beginning the experiment. If at any point during the study you wish to cease participating, please simply stop and exit the browser window. You do not need to contact the experimenters if you decide to withdraw. If after participating you wish to have your data omitted from analysis, please contact the experimenters at the xxyyzzy@ucl.ac.uk.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="consentCheck">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>300</y>
       <width>550</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>I have read the information above and I consent to take part in this study.</string>
     </property>
     <property name="iconSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="dataCheck">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>380</y>
       <width>850</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>I understand that my data may be used in future academic papers and publications without my knowledge. </string>
     </property>
     <property name="iconSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="withdrawCheck">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>460</y>
       <width>850</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>I understand that I can withdraw from participation by closing the window.</string>
     </property>
     <property name="iconSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="ageCheck">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>540</y>
       <width>850</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>I confirm that I am over the age of 18.</string>
     </property>
     <property name="iconSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="continueButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>650</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="settingsBox">
     <property name="geometry">
      <rect>
       <x>1216</x>
       <y>70</y>
       <width>291</width>
       <height>231</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>171</red>
           <green>171</green>
           <blue>171</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>171</red>
           <green>171</green>
           <blue>171</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>171</red>
           <green>171</green>
           <blue>171</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>171</red>
           <green>171</green>
           <blue>171</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="login">
      <widget class="QLineEdit" name="pwBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>301</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Please enter the password to access settings.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="logButton">
       <property name="geometry">
        <rect>
         <x>115</x>
         <y>150</y>
         <width>70</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="settings" native="true">
      <layout class="QFormLayout" name="formLayout">
       <property name="horizontalSpacing">
        <number>8</number>
       </property>
       <property name="verticalSpacing">
        <number>12</number>
       </property>
       <item row="0" column="0">
        <widget class="QSpinBox" name="cond1">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="cond1lab">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Condition 1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="cond2">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="cond2lab">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Condition 2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QSpinBox" name="cond3">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="cond3lab">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Condition 3</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QSpinBox" name="cond4">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="cond4lab">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Condition 4</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="saveButton">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>SAVE</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="enableButton">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ADD 4TH</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="disableButton">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>REMOVE 4TH</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="settingsButton">
     <property name="geometry">
      <rect>
       <x>1429</x>
       <y>20</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>settings</string>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="demographics">
    <widget class="QLabel" name="demHeader">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>1600</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Please respond to all of the following demographic questions. </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameInput">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>240</y>
       <width>350</width>
       <height>45</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="name">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>245</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Name:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="age">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>345</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Age:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Gender">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>445</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Gender:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="sexGroup">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>410</y>
       <width>500</width>
       <height>100</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QRadioButton" name="maleButton">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>25</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Male</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="femaleButton">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>25</y>
        <width>131</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Female</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="otherButton">
      <property name="geometry">
       <rect>
        <x>245</x>
        <y>25</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Other</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="verifyButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>650</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
    <widget class="QLabel" name="education">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>540</y>
       <width>371</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Are you currently in full time education?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="educCheck">
     <property name="geometry">
      <rect>
       <x>415</x>
       <y>540</y>
       <width>51</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
    </widget>
    <widget class="QSpinBox" name="ageSpin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>345</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="demSubHead">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>1600</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Note that information gathered here is purely for research purposes. Any data used in future publication will be completely anonymised.</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="briefPage">
    <widget class="QLabel" name="briefLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>-150</y>
       <width>1400</width>
       <height>600</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Consider the following problem carefully, then respond with your decision. &lt;/p&gt;&lt;p&gt;Here are two urns, labeled A and B, containing red and blue marbles, and you have to draw a marble from one of the urns. If you get a blue marble, you will be entered into a £30 lottery draw. Urn A contains 50 red marbles and 50 blue marbles. Urn B contains marbles in an unknown color ratio, from 100 red marbles and 0 blue marbles to 0 red marbles and 100 blue marbles. The mixture of red and blue marbles in Urn B has been decided by writing the numbers 0, 1, 2, . . ., 100 on separate slips of paper, shuffling the slips thoroughly, and then drawing one of them at random. The number chosen was used to determine the number of blue marbles to be put into Urn B, but you do not know the number. &lt;/p&gt;&lt;p&gt;Every possible mixture of red and blue marbles in Urn B is equally likely. You have to decide whether you prefer to draw a marble at random from Urn A or Urn B. What you hope is to draw a blue marble and be entered for the £30 lottery draw. Consider very carefully from which urn you prefer to draw the marble, then choose your decision below by clicking on the urn. Your choice will be locked in and A marble will be drawn from your chosen urn as soon as you click the &amp;quot;Draw&amp;quot; button.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="urnA">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>300</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Urn A</string>
     </property>
    </widget>
    <widget class="QLabel" name="urnB">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>300</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Urn B</string>
     </property>
    </widget>
    <widget class="QPushButton" name="drawButton">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>700</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Draw</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="debriefPage">
    <widget class="QLabel" name="debriefHeader">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>225</y>
       <width>1600</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>The experiment is now over. Thank you for your participation.</string>
     </property>
    </widget>
    <widget class="QLabel" name="debriefHeader_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>295</y>
       <width>1000</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>This experiment aims to understand whether people are &quot;uncertainty averse&quot; - in other words, will they prefer a known chance of winning over an unknown one.</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="debriefHeader_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>425</y>
       <width>1600</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>You may now return to the experimenter, where you may ask any questions you have and receive your payment.</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="condition_100"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
